src/__tests__/forms.spec.js

```diff
@@ -217,19 +217,50 @@ describe('Unit Test cases for use-form hook', () => {
       expect(result.current[0].name.isValid).toBe(true);
       expect(result.current[1].isValid).toBe(true);
     });
-    it('should validate field having async functions throws error', async () => {
+    it('should validate field when empty value is set', async () => {
       const { result, waitForNextUpdate } = renderHook(() =>
         useForm({
           name: {
             validationFns: [
-              jest.fn().mockRejectedValueOnce(new Error('Async error')),
+              (nameValue, formValues) => [
+                formValues.name && formValues.name.startsWith('a'),
```

Comment:
can we use formValues.name?.startsWith('a')

Reply:
would have to setup babel for that. i